app/scripts/directives/podMetrics.js

```diff
@@ -130,6 +146,18 @@ angular.module('openshiftConsole')
         // Show last hour by default.
         scope.options.timeRange = _.head(scope.options.rangeOptions);
 
+        function generateVolumeDataSets() {
+          angular.forEach(scope.pod.spec.volumes, function(volumeData) {
+            if (volumeData.persistentVolumeClaim !== undefined) {
```

Comment:
In my opinion, the volumeDataSets will only include persistent Volumes which "persistentVolumeClaim" can identify.